docs/HowToFAQ/FAQParallel.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Does Dafny verify methods in parallel?
+---
+
+## Question
+
+Does Dafny verify methods in parallel?
+
+## Answer
+
+For the most part, Dafny verifies each method in each module sequentially
+and attempts to prove all the assertions in a method in one go.
+However, you can use the option `/vcsCores:1` to parallelize some of the activity
```

Comment:
Wouldn't you want a number greater than `1`?

Reply:
Revised